File: DS/Java/Vector/Main_Example.java
import java.util.Vector;

public class Main {
    public static void main(String[] args) {
        Vector<String> vector = new Vector<>();
        
        // Adding elements
        vector.add("First");
        vector.add("Second");
        vector.add("Third");
        
        // Adding element at specific index
        vector.add(1, "Between");
        
        // Accessing elements
        System.out.println("Element at index 1: " + vector.get(1));
        
        // Size and capacity
        System.out.println("Size: " + vector.size());
        System.out.println("Capacity: " + vector.capacity());
        
        // Removing elements
        vector.remove("Second");
        System.out.println("After removal: " + vector);
    }
}

File: DS/Golang/List.List/main_example.go
package main

import (
    "container/list"
    "fmt"
)

func main() {
    // Create a new list
    l := list.New()
    
    // Push elements to back
    l.PushBack("first")
    l.PushBack("second")
    l.PushBack("third")
    
    // Push elements to front
    l.PushFront("zero")
    
    // Iterate through list
    fmt.Println("Forward iteration:")
    for e := l.Front(); e != nil; e = e.Next() {
        fmt.Println(e.Value)
    }
    
    // Reverse iteration
    fmt.Println("\nReverse iteration:")
    for e := l.Back(); e != nil; e = e.Prev() {
        fmt.Println(e.Value)
    }
    
    // Remove element
    element := l.Front().Next()
    l.Remove(element)
    
    fmt.Println("\nAfter removing second element:")
    for e := l.Front(); e != nil; e = e.Next() {
        fmt.Println(e.Value)
    }
}
